templates/receipt/templates/main.tpl

```diff
@@ -0,0 +1,33 @@
+{{define "simple-paragraph"}}
+    <paragraph>
+        <text-chunk font="times" font-size="9">{{.}}</text-chunk>
+    </paragraph>
+{{end}}
+{{define "table-cell-paragraph"}}
+    <table-cell align="left" border-style="none" border-width="0">
+        {{template "simple-paragraph" .}}
+    </table-cell>
+{{end}}
+<division margin="0 10 10 10" padding="5, 5, 5, 5">
```

Comment:
The `margin` and `padding` attributes support shorthand syntax. So if all values are `5`, you can define the padding as `padding="5"`.